volttron/utils/rmq_mgmt.py

```diff
@@ -548,8 +498,9 @@ def create_exchange(self, exchange, properties, vhost=None, ssl_auth=None):
         :param ssl_auth: Flag for SSL connection
         :return:
         """
-        vhost = vhost if vhost else self.get_vhost()
-        _log.debug("Create new exchange: {}".format(exchange))
+        ssl_auth = ssl_auth if ssl_auth is not None else self.is_ssl
+        vhost = vhost if vhost else self.rmq_config.virtual_host
```

Comment:
vhost is always from the rabbitmq_config.yml.  So we should get rid of vhost parameter in all the method and just use rmq_config.vhost. That way we don't need this vhost= if vhost else... statement..

Reply:
Leaving it for now